Multisig/Logic/Models/Transaction/AddressString.swift

```diff
@@ -24,7 +24,7 @@ struct AddressString: Hashable, Decodable {
         let container = try decoder.singleValueContainer()
         let value = try container.decode(String.self)
         do {
-            address = try Address.init(hex: value, eip55: false)
+            address = try Address(from: value)
```

Comment:
Do we need to have "from"?

Reply:
Yes, because there are other constructors that use 'value' but do not throw, I wanted to have a distinct label for a throwing constructor